packages/faucet/src/faucet.ts

```diff
@@ -89,8 +85,13 @@ export class Faucet {
    * Throws an error if the transaction failed.
    */
   public async send(job: SendJob): Promise<void> {
-    const result = await this.clients[job.sender].sendTokens(job.recipient, [job.amount], constants.memo);
-    assertIsBroadcastTxSuccess(result);
+    const client = this.clients[job.sender];
+    if (client instanceof CosmosClient) {
+      const result = await client.sendTokens(job.recipient, [job.amount], constants.memo);
```

Comment:
Impressive that TS understands that if `client` is `CosmosClient`, it also is `SigningCosmosClient`.

But since `CosmosClient` has no `sendTokens`, I'd test `client instanceof SigningCosmosClient`